app/components/avo/turbo_frame_wrapper_component.html.erb

```diff
@@ -1,10 +1,13 @@
-<% if @name.present? %><turbo-frame id="<%= @name %>"><% end %>
-<%
-  # When rendering the frames the flashed content gets lost.
-  # We're appending it back if it's a turbo_frame_request.
-%>
-<% if helpers.turbo_frame_request? %>
-  <%= helpers.turbo_stream_action_tag :append, target: "alerts", template: render(Avo::FlashAlertsComponent.new(flashes: helpers.flash)) %>
+<% if @name.present? %>
+  <turbo-frame id="<%= @name %>">
+    <% if helpers.turbo_frame_request? %>
+      <%= helpers.turbo_stream_action_tag :append, target: "alerts", template: render(Avo::FlashAlertsComponent.new(flashes: helpers.flash)) %>
+    <% end %>
+    <%= content %>
+  </turbo-frame>
+<% else %>
+  <% if helpers.turbo_frame_request? %>
+    <%= helpers.turbo_stream_action_tag :append, target: "alerts", template: render(Avo::FlashAlertsComponent.new(flashes: helpers.flash)) %>
+  <% end %>
+  <%= content %>
```

Comment:
It is more verbose but it seems the first syntax is not currently supported by Herb.

Reply:
We can go with this duplication for now, I think the previous implementation was a bit hacky